warehouse/dbt_project.yml

```diff
@@ -48,6 +48,16 @@ models:
       +materialized: table
       schema: views
 
+    # To-do: should probably end up in `staging`
+    payments_views_staging:
```

Comment:
**suggestion:** When there's no inherent ordering to a list (like this list of schemas) I suggest ordering things alphabetically. It removes some of the mental work necessary to scan the list.

I see that the `marts` schema already violates this, so I don't know whether an ordering has previously been discussed.